### PR TITLE
add kubebuilder prerequisites in contributing.md

### PR DESCRIPTION
**Pull request by qingfengming**
case offical doc only show v3 installation, add the v2 installation here.

[kubebuilder quick start, install](https://book.kubebuilder.io/quick-start.html#installation)

[discussion](https://github.com/oam-dev/kubevela/discussions/1672)

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -17,8 +17,28 @@ contributing to `kubevela` or build a PoC (Proof of Concept).
...
+5. kubebuilder v2.X.X
 
-We also recommend you to learn about KubeVela's [design](https://kubevela.io/docs/concepts) before dive into its code.
+##### kubebuilder install:
+
+linux:
+```
+wget https://github.com/kubernetes-sigs/kubebuilder/releases/download/v2.0.0-alpha.1/kubebuilder_2.0.0-alpha.1_linux_amd64.tar.gz
+tar -zxvf  kubebuilder_2.0.0-alpha.1_linux_amd64.tar.gz
+sudo mv kubebuilder_2.0.0-alpha.1_linux_amd64 /usr/local/kubebuilder
```
Comment (captainroy-hy) [inline CONTRIBUTING.md:28]: sudo mv kubebuilder_2.0.0-alpha.1_linux_amd64/**bin** /usr/local/kubebuilder

Comment (captainroy-hy) [reply CONTRIBUTING.md:28]: k8s control plane bin in kubebuilderv2.0.0 bundle is v1.14.1, but kubevela requires  k8s v1.15+.  
Maybe you can change it to kubebuilder v2.3.1 which is used in KubeVela CI.

**Review by captainroy-hy — COMMENTED** [review]
Good suggestion!

File: CONTRIBUTING.md (hunk 1 above)
Comment (qingfengming) [reply CONTRIBUTING.md]: thanks, i will recommit once.

**Pull request closed without merging**